### ISSUE TITLE
landuse=brownfield looks fairly similar to buildings

### ISSUE THREAD

**Issue by imagico** [gravitystorm/openstreetmap-carto#2353]
See for example here:

![landuse=brownfield](http://www.imagico.de/files/brownfield.png)

http://overpass-turbo.eu/s/isQ

**Comment by imagico** [comment]
Other lower zoom level example:

http://overpass-turbo.eu/s/isX

**Comment by kocio-pl** [comment]
I don't see the problem in the given example at this zoom level. I can see it for lower zoom levels, but I think it would be hard to find another color which is more distinct, yet not as dominant as before [the change](https://github.com/gravitystorm/openstreetmap-carto/pull/2292), which was a real problem for me. Lower zoom levels don't have to show all the differences, rather general shape, and current solution is good enough for me in that regard.

**Comment by pnorman** [comment]
👍 for it being an issue, but finding a suitable colour is always a pain.

**Comment by imagico** [comment]
Yes, IMO #2292 did not improve things overall but this is a tough call.  Map clarity is important and in general i much prefer not rendering something to rendering it in a misleading or confusing way.

Addressing this in an elegant way will require a look at the bigger picture, looking at both construction areas and roads (and other lifecycle related features) and looking at industrial landuses in the strict sense (i.e. landuse=quarry, landuse=landfill).

**Comment by kocio-pl** [comment]
I have the similar problem, which is much more visible for me (but still not important enough to open a separate ticket): landuse=commercial and landuse=retail are too similar. I can recognize them properly only if both such areas are close, otherwise I'm not sure in many cases.

**Comment by imagico** [comment]
The difference is landuse=commercial/retail are very similar with many boundary cases even while buildings and landuse=brownfield are something completely different.

**Comment by kocio-pl** [comment]
Related to https://github.com/gravitystorm/openstreetmap-carto/issues/1991.

**Comment by Tomasz-W** [comment]
I can admit that colour of brownfield is similar to buildings, but I think it isn't too similar and I don't see a big problem here. Note that buildings have outlines and brownfields haven't. There was no ideas how to solve it for a 2 years.
@kocio-pl What about closing it as "won't fix"?

**Comment by kocio-pl** [comment]
You're right probably - on the individual buildings level I can perfectly recognize them, at lower zoom levels I don't think it's important. Please reopen if further discussion is needed.

**Comment by jeisenbe** [comment]
I think this issue should be re-opened. 

This problem  is especially visible at z14 where buildings have no outlines and are darker:
`@building-low-zoom: darken(@building-fill, 4%);` is `#cfc4b9` Lch(80,6,71) - this is rather close to `@construction: `#c9c4b4` Lch(80,10,109) - the lightness is identical, the chroma are almost the same, and the hue difference can be hard to see when the background is different. The delta is 6.5. 

For example, buildings on industrial color background look very similar to adjacent construction areas:

@imagico has suggested the brownfield / construction color could be changed and this might fix the problem. For example, `#c5bea6` would have a delta of 8.8, and it's lightness is 77, so this alone would make it less similar to low-zoom buildings. 

Another option would be to reconsider the low-zoom buildings color. It is only used at this one zoom level, z14, where the outlines are not visible.

**Comment by Adamant36** [comment]
You should make a test of @imagico's color. It might worth changing even if the low-zoom building color is changed.

**Comment by imagico** [comment]
Reopening because this has been closed essentially as subjective wontfix without consensus.
